src/briefcase/commands/dev.py

```diff
@@ -110,6 +110,37 @@ def install_dev_requirements(self, app: AppConfig, **options):
         else:
             self.console.info("No application requirements.")
 
+    def create_environment(self, app: AppConfig, **options):
+        # This path is defined for a venv style environment (currently the default environment manager used)
+        venv_path = self.base_path / ".briefcase" / f"dev-web-venv-{app.app_name}"
```

Comment:
A minor organization thing - it might be worth creating an `app.app_name` folder, and creating the venv under that. Right now, it won't be a problem, but over time, the `.briefcase` folder is going to gain more content; having that content organised by app would be helpful.